pkg/controllers/netpol/network_policy_controller.go

```diff
@@ -817,6 +817,14 @@ func (npc *NetworkPolicyController) syncPodFirewallChains(version string) (map[s
 			}
 		}
 
+		// add rule to log the packets that will be droopped due to network policy enforcement
+		comment = "rule to log dropped traffic POD name:" + pod.name + " namespace: " + pod.namespace
```

Comment:
```suggestion
		comment = "rule to log dropped traffic to POD name:" + pod.name + " namespace: " + pod.namespace
```

Reply:
@mrueg thanks for the review. I started with same but actually a only a single IPtable rule gets created so unfortunaltye can not add to/from. Resolved other comment